packages/icons/icons.json

```
@@ -933,217 +933,217 @@
         "codepoint": 61883
     },
     {
         "displayName": "Social media",
         "iconName": "social-media",
-        "tags": "circle, rotate, share",
+        "tags": "circle, rotate, share, harmony, tadpoles, circuit, in progress, run, loop, cycle",
         "group": "action",
         "codepoint": 62144
     },
     {
         "displayName": "Arrow: up",
         "iconName": "arrow-up",
-        "tags": "direction, north",
+        "tags": "direction, north, up arrow, arrow, up, upwards",
         "group": "interface",
         "codepoint": 61737
     },
     {
         "displayName": "Arrow: down",
         "iconName": "arrow-down",
-        "tags": "direction, south",
+        "tags": "direction, south, down, down arrow, please see below, descend, scroll down",
         "group": "interface",
         "codepoint": 61732
     },
     {
         "displayName": "Arrows: horizontal",
         "iconName": "arrows-horizontal",
-        "tags": "direction, level",
+        "tags": "direction, level, arrows, bidirectional, left-right, resize, move horizontally, swap, arrow, switch, link, reaction, alternate",
         "group": "interface",
         "codepoint": 61738
     },
     {
         "displayName": "Arrows: vertical",
         "iconName": "arrows-vertical",
-        "tags": "direction, level",
+        "tags": "direction, level, arrows, two way, up down, drag up or down, expand, collaps, open, up and down, up down arrow",
         "group": "interface",
         "codepoint": 61739
     },
     {
         "displayName": "Resolve",
         "iconName": "resolve",
-        "tags": "circles, divide, split",
+        "tags": "circles, divide, split, intersection, overlap, venn diagram",
         "group": "action",
         "codepoint": 62099
     },
     {
         "displayName": "Graph",
         "iconName": "graph",
-        "tags": "graph, diagram",
+        "tags": "diagram, chemistry, molecule, directed graph, linked entities, relational elements",
         "group": "data",
         "codepoint": 61909
     },
     {
         "displayName": "Briefcase",
         "iconName": "briefcase",
-        "tags": "suitcase, business, case, baggage,",
+        "tags": "suitcase, business, case, baggage, , work, kit, toolbox",
         "group": "miscellaneous",
         "codepoint": 61753
     },
     {
         "displayName": "Dollar",
         "iconName": "dollar",
-        "tags": "currency, money",
+        "tags": "currency, money, dollar sign",
         "group": "miscellaneous",
         "codepoint": 61825
     },
     {
         "displayName": "Ninja",
         "iconName": "ninja",
-        "tags": "star, fighter, symbol",
+        "tags": "star, fighter, symbol, vortex, shuriken, throwing star, ninja star",
         "group": "miscellaneous",
         "codepoint": 62042
     },
     {
         "displayName": "Delta",
         "iconName": "delta",
-        "tags": "alt j, symbol",
+        "tags": "alt j, symbol, triangle, diff",
```

```suggestion
        "tags": "symbol, triangle, diff, change, formula, greek, alphabet",
```

while we're here, we should really remove "alt j" 🤦🏽

---

haha that's hilarious